program_intent_engagement/apps/core/models.py

```diff
@@ -52,7 +52,7 @@ class ProgramIntent(TimeStampedModel):
         ('MAYBE', 'MAYBE'),
     )
 
-    user = models.ForeignKey(User, db_index=True, on_delete=models.CASCADE)
+    lms_user_id = models.IntegerField(null=True, db_index=True)
```

Comment:
ahh that was something I copied over from the user model, this field shouldn't be nullable so I will update.

Reply:
In order to make this field nullable, I first had to add a default value, and then remove the default. I believe this is fine because there is no data for this model in stage or prod.